x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/form/serializer/serializer.ts

```diff
@@ -60,9 +60,16 @@ export const createSerializer = (originalPolicy?: SerializedPolicy) => (
         if (_meta.hot.bestCompression && hotPhaseActions.forcemerge) {
           hotPhaseActions.forcemerge.index_codec = 'best_compression';
         }
+
+        if (_meta.hot.readonlyEnabled) {
+          hotPhaseActions.readonly = hotPhaseActions.readonly ?? {};
```

Comment:
Won't this always equal `{}`, or is it possible to pass options to the `readonly` action?

Reply:
Currently, the `readonly` action is always empty, but I thought we should not overwrite the existing object in case ES will add any new properties.